### PR TITLE
Added div and test function

### PR DESCRIPTION
**Pull request by alekhyamandali**
Hi @rclif 

I have created a new branch for divide. please have a look at it

Thanks

### CODE REVIEW

File: divide.py
Comment (rclif) [inline divide.py:1]: Hi @alekhyamandali 

I think the "from.divide import divide" line should be in the "pythoncalculator/__init__.py" file

File: divide.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from .divide import divide
```
Comment (rclif) [inline divide.py]: I think the "from.divide import divide" line should be in the "pythoncalculator/init.py" file

File: pythoncalculator/divide.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from .divide import divide
```
Comment (iainwebb) [inline pythoncalculator/divide.py]: I don't think this line should be here.

**Review by iainwebb — COMMENTED** [review]
I've spotted another error.

**Review by rclif — COMMENTED** [review]
Hi @alekhyamandali ,

I think there is an error in the divide.py file

**Review by rclif — APPROVED** [review]
Looks good to merge

**Pull request merged**